Kitodo-XML-SchemaConverter/src/main/resources/xslt/marc21slim2mods.xsl

```diff
@@ -0,0 +1,5508 @@
+<xsl:stylesheet xmlns="http://www.loc.gov/mods/v3" xmlns:marc="http://www.loc.gov/MARC21/slim"
+	xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+	exclude-result-prefixes="xlink marc" version="1.0">
+	<xsl:include href="./src/main/resources/xslt/MARC21slimUtils.xsl"/>
```

Comment:
Should the reference in `href` not contain only a relative path from the current file as both xsd files are in the same directory?

Reply:
I tried different relative path variations (`MARC21slimUtils.xsl`, `./MARC21slimUtils`, `.MARC21slimUtils` ...) but always got an error akin to
```Caused by: java.io.FileNotFoundException: /Users/arved/Projects/Kitodo/kitodo-production/Kitodo-XML-SchemaConverter/MARC21slimUtils.xsl (No such file or directory)```
The path variant I ended up with is the only one which worked and did not cause an error.

Reply:
I tried it in the last hour too but I did not even get this working. Maybe we found a better way in future.